src/libraries/System.Security.Cryptography.Algorithms/ref/System.Security.Cryptography.Algorithms.cs

```diff
@@ -483,6 +483,7 @@ public partial class HMACSHA1 : System.Security.Cryptography.HMAC
         public HMACSHA1() { }
         public HMACSHA1(byte[] key) { }
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        [System.ObsoleteAttribute("HMACSHA1 always uses the algorithm implementation provided by the platform. Use the constructor without the useManagedSha1 parameter.", DiagnosticId = "SYSLIB0030", UrlFormat = "https://aka.ms/dotnet-warnings/{0}")]
```

Comment:
"Use the constructor" makes it sound like there's only one.  Should it be "use _a_ constructor"?

Reply:
Well, there is only one other one that accepts the `key` parameter which I think is where my mind was going with that. But yeah, there's no point in the language limiting it like there is only one (maybe another will get added later).